dns/zone-configs/x-k8s.io._0_base.yaml

```diff
@@ -1,3 +1,4 @@
 # The top-level x-k8s.io zone
-
-# Nothing here.
+'':
```

Comment:
heh, it was a long time since I touch DNS, do we want a wildcard here? 
https://github.com/octodns/octodns/blob/master/docs/records.md#names